<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarFondosDialog</class>
 <widget class="QDialog" name="AgregarFondosDialog">
  <property name="windowTitle">
   <string>Añadir Fondos</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cantidad a añadir (€):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_cantidad"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_agregar">
       <property name="text">
        <string>Pagar y Añadir saldo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
